locale/CMakeLists.txt

```diff
@@ -1,12 +1,19 @@
 if (LEATHERMAN_USE_LOCALES)
     find_package(Boost 1.54 REQUIRED COMPONENTS locale system)
+    if (BOOST_STATIC AND LEATHERMAN_USE_ICU)
+        find_package(ICU)
```

Comment:
Ah, thanks for the feedback! I hadn't actually tried using it, always the best way to add code.